src/MessagePack.UnityClient/Assets/Scripts/MessagePack/MessagePackWriter.cs

```diff
@@ -1290,6 +1290,91 @@ private unsafe void WriteString_PostEncoding(byte* pBuffer, int estimatedOffset,
             }
         }
 
+        /// <summary>
+        /// Get the encoded byte length of the given integer.
+        /// </summary>
+        /// <param name="value">The integer to encode.</param>
+        /// <returns>Encoded length, either 1, 2, 3, 5 or 9 bytes.</returns>
+        public static int IntegerEncodedLength(long value)
```

Comment:
I'm not a big fan of the added MessagePackWriter methods, although maybe they can be generally useful. I'd rename them at least. And maybe just mark them as `internal` for now. We'd want to add tests for them too.

Reply:
I tried to mark them as `internal`, but the compiler emitted an error in the call site (I forgot what the actual error was, but it's probably about not being able to access the method).


I added tests for this methods inside the serialization tests for integer types, since they already have checks about how many bytes each integer should serialize to, so I can reuse the numbers. Do I need a separate test just for this method?